lib/vsc/filesystem/oceanstor.py

```diff
@@ -1703,12 +1703,6 @@ def set_fileset_quota(self, soft, fileset_path, fileset_name=None, hard=None, in
         quota_limits = {"soft": soft, "hard": hard, "inode_soft": inode_soft, "inode_hard": inode_hard}
         self._set_quota(who=fileset_name, obj=fileset_path, typ="fileset", **quota_limits)
 
-        # TODO: remove once OceanStor supports creating user quotas on non-empty filesets
```

Comment:
is this fixed? Or why can it be removed?

Reply:
I still see this happening in the logs as early as May 7th:
> 2025-05-07 09:13:52,188 WARNING    OceanStorOperations MainThread  Creation of user quota for 'vscXXXXX' in '00@0000' failed, but a default quota should be in place. Moving on.

Reply:
So why are you removing it? :upside_down_face:

Reply:
ohh wait, we are talking about two different things.

My comment is about line 1828 which prints the aforementioned warning and indeed cannot be removed and it is not removed; because oceanstor still fails to make user quotas on non-empty filesets.

Your comment is about line 1706; which exists due to the same issue in oceanstor about non-empty quotas. It is removed because it is redundant, the default quota is already handled in `make_fileset` line 1336, just after calling this `set_fileset_quota` method in line 1320.